test_get_screen_lines() sometimes fail because no line is long enough

`test_get_screen_lines()` sometimes fail with following message

```
    def test_get_screen_lines():
        loadbar = px_load_bar.PxLoadBar(1, 1)
        baseline = px_process.get_all()
    
        SCREEN_ROWS = 10
        SCREEN_COLUMNS = 100
        lines = px_top.get_screen_lines(
            loadbar, baseline, SCREEN_ROWS, SCREEN_COLUMNS)
    
        # Top row should contain ANSI escape codes
        CSI = b"\x1b["
        assert b'CSI' in lines[0].replace(CSI, b'CSI')
    
        assert len(lines) == SCREEN_ROWS
    
        # This is a bit fluffy, but due to ANSI escape codes the line lengths are
        # difficult to measure. So if at least one line is exactly the right length
        # then we're probably fine.
        found_right_length = False
        for line in lines:
            if len(line) == SCREEN_COLUMNS:
                found_right_length = True
>       assert found_right_length
E       assert False

tests/px_top_test.py:102: AssertionError
```
https://github.com/walles/px/blob/408cc5fb7c59856750eb8c9640a31fbc41e143da/tests/px_top_test.py#L80

as far as i understand this test takes the lines from the actual processes on my system. then takes the first ten lines. then checks to see whether at least one of the ten lines is at least `SCREEN_COLUMNS` long. that fails when the processes on my system do not have a line that long.

instead of calling `px_process.get_all()` it would be better to provide `baseline` with a hardcoded literal containing known good values. this test is about `px_top.get_screen_lines()` and should not depend on the output of `px_process.get_all()` from the live system.

--- a/tests/px_top_test.py
+++ b/tests/px_top_test.py
@@ -3,6 +3,7 @@ import os
 from px import px_top
 from px import px_process
 from px import px_load_bar
+from px import px_terminal
 
 import testutils
 
@@ -82,7 +83,7 @@ def test_get_screen_lines():
     baseline = px_process.get_all()
 
     SCREEN_ROWS = 10
-    SCREEN_COLUMNS = 100
+    SCREEN_COLUMNS = 70
     lines = px_top.get_screen_lines(
         loadbar, baseline, SCREEN_ROWS, SCREEN_COLUMNS)
 
@@ -92,11 +93,12 @@ def test_get_screen_lines():
 
     assert len(lines) == SCREEN_ROWS
 
-    # This is a bit fluffy, but due to ANSI escape codes the line lengths are
-    # difficult to measure. So if at least one line is exactly the right length
-    # then we're probably fine.
-    found_right_length = False
-    for line in lines:
-        if len(line) == SCREEN_COLUMNS:
-            found_right_length = True
-    assert found_right_length
+    # Row three is the heading line, it should span the full width of the screen
+    # and be in inverse video.
+    assert len(lines[2]) == len(px_terminal.inverse_video('x' * SCREEN_COLUMNS))
+
+    # The actual process information starts at line four
+    for line in lines[3:]:
+        # No line can be longer than the screen width; long lines should have
+        # been cut
+        assert len(line) <= SCREEN_COLUMNS

